dbt_project.yml

```diff
@@ -160,10 +161,18 @@ on-run-end:
 models:
   snowplow_unified:
     +materialized: table
-    +file_format: delta
+    +file_format: >
```

Comment:
I know we use vars in model configs but never in the project.yml so not sure how safe it is. Is this standard dbt encouraged usage, do you know? Just wondering if it gets properly compiled when overwritten in a project for instance. Also not sure if this would always get evaluated to none if it gets processed from top to bottom. If we are not sure, better get this tested, though most likely it will only turn to a potential bug if we add more formats.